### DISCUSSION TITLE
Saving data

### DISCUSSION THREAD

**Discussion by Rasmus-Petersen-RLL-Sampension** [Innofactor/XrmToolBox.BulkDataUpdater#164] [Ideas]
BDU would be the perfect tool to anonymize data if it was possible to save ones settings.

So generate a document containing:
* Fetch used to retrieve records.
* For Update - Attributes to be updated and Execution settings
* For Assign - The user/team to assign and Execution settings
* For Set State - The status and status reason and Execution settings
* For Delete - The Execution settings

And then of course functionality to load any previous saved settings.

Such an UPGRADE will come in extremely handy when one lives on the blessings of the EU GDPR rules and one therefore should anonymize any data in DEV and TEST environments.

**Comment by rappen** [comment]
That's a damn good idea!
Thanks - I will start to work on this "soon"...
